packages/plugin-compiler/src/main/kotlin/io/realm/kotlin/compiler/RealmModelLoweringExtension.kt

```diff
@@ -57,6 +57,10 @@ private class RealmModelLowering(private val pluginContext: IrPluginContext) : C
             if (irClass.isData) {
                 error("Data class '${irClass.kotlinFqName}' is not currently supported.")
             }
+            // We don't support enum class
+            if (irClass.isEnumClass) {
+                error("Enum class '${irClass.kotlinFqName}' is not currently supported.")
```

Comment:
```suggestion
                error("Enum class '${irClass.kotlinFqName}' is not supported. Only normal classes can inherit from 'RealmObject' or 'EmbeddedRealmObject'.")
```
I know this sentence was just copied from the data class above, but IMO we should rewrite it to something like what I proposed. It doesn't promise anything about future compatibility + point to what is the correct solution.

Reply:
That makes sense, I wanted to suggest that when I noticed the word "currently"